web/themes/joinup/joinup_theme.theme

```diff
@@ -116,23 +142,70 @@ function joinup_theme_preprocess_fieldset(&$variables) {
 }
 
 /**
+ * Implements hook_preprocess_HOOK() for container.html.twig.
+ */
+function joinup_theme_preprocess_container(&$variables) {
+  $route_match = \Drupal::routeMatch();
+  $route_name = $route_match->getRouteName();
+  $form_routes = array(
+    'node.add',
+    'rdf_entity.rdf_add',
+    'entity.node.edit_form',
+    'collection.propose_form',
+    'entity.rdf_entity.edit_form',
+  );
+
+  // Check if container is field element - it has only one parent.
+  if (!empty($route_name) && in_array($route_name, $form_routes)
+    && isset($variables['element']['#parents'])
+    && count($variables['element']['#parents']) == 1
+    && $variables['element']['#id'] != 'edit-actions') {
+    $variables['is_form_wrapper'] = TRUE;
+  }
+}
+
+/**
  * Implements hook_theme_suggestions_alter().
  */
 function joinup_theme_theme_suggestions_alter(array &$suggestions, array $variables) {
-  if (isset($variables['element']) && isset($variables['element']['#type'])) {
+  if (isset($variables['element']['#type'])) {
+    $original_theme_hook = $variables['theme_hook_original'];
+    $floating_label_types = array(
+      'textfield',
+      'url',
+      'password',
+      'email',
+      'entity_autocomplete',
+    );
+
     if ($variables['element']['#type'] == 'form') {
-      $original_theme_hook = $variables['theme_hook_original'];
       $suggestions[] = $original_theme_hook . '__' . str_replace('-', '_', $variables['element']['#id']);
```

Comment:
I see that you removed this part of the code. Anyway, the `form_id` will always be there, it's set in the form builder. And the id will be modelled after it.